juan-pablo-hernandez/tarea-1/style.css

```
@@ -0,0 +1,76 @@
+* {
+    margin: 0;
+}
+body {
+    overflow: hidden;
+    background-color: #e4e0e0;
+    font-family: 'Fira Sans Condensed', sans-serif;
+}
+.magazine-container {
+    width: 100vw;
+    height: 100vh;
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+.magazine-cover {
+    height: calc(80vh - 100px);
+    width: auto;
+    padding: 40px 20px;
+    aspect-ratio: 5 / 7;
+    border: 50px solid #ffce00;
+    background-image: url('/src/magazine-bg.jpg');
```

@juan-hm , please use relative paths
```suggestion
    background-image: url('src/magazine-bg.jpg');
```